scripts/linters/general_purpose_linter.py

```diff
@@ -409,27 +402,12 @@
         'excluded_files': (),
```

Comment:
Move this check to the SingleLinePragamChecker.

Reply:
Is there something I need to do for this comment, or is this a future plan?